k8s_authorization/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+k() {
+    kubectl --context kind-opa-authorizer "$@"
+}
+
+pwd=$(pwd) envsubst < config/kind-conf.yaml | kind create cluster --name opa-authorizer --config -
```

Comment:
https://linux.die.net/man/1/envsubst this one? Interesting. I haven't had it installed before.

Reply:
Yep, that's the one! Awesome little utility function.. though I must admit I thought it was installed by default. Apparently not though, so I'll rewrite this to remove that dependency.